styles/components/_dictionary.scss

```diff
@@ -22,6 +21,7 @@
 }
 
 .sprk-c-Dictionary .sprk-b-Label {
+  color: $sprk-dictionary-label-color;
```

Comment:
Needed to add this so Dictionary wouldn't pick up the new Input Label color.

Reply:
Okay, in future, we'll want to add a new class for labels in dictionaries if they are meant to be different. We can still use the label class and get what we need as building blocks but then add a dictionary label class that sets the color that's specifically for dictionary.